src/components/DataView/DataView.js

```diff
@@ -282,36 +295,164 @@ const DataView = React.memo(function DataView({
+            fields={preparedFields}
+            hasAnyExpansion={hasAnyExpansion}
+            onSelect={toggleEntrySelect}
+            onSelectAll={selectAll}
+            renderSelectionCount={renderSelectionCount}
+            rowHeight={tableRowHeight}
+            selectable={canSelect}
+          />
+        ) : (
+          <TableView
+            alignChildOnField={Math.min(
+              Math.max(-1, alignChildOnField),
+              fields.length - 1
+            )}
+            allSelected={allSelected}
+            entries={renderedEntries}
+            fields={preparedFields}
+            hasAnyActions={hasAnyActions}
+            hasAnyExpansion={hasAnyExpansion}
+            onSelect={toggleEntrySelect}
+            onSelectAll={selectAll}
+            renderSelectionCount={renderSelectionCount}
+            rowHeight={tableRowHeight}
+            selectable={canSelect}
+            selectedCount={selectedIndexes.length}
+          />
+        ))}
+
+      {emptyEntries && (
+        <div
+          css={`
+            height: ${85 * GU}px;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+          `}
+        >
+          <div
+            css={`
+              width: ${31 * GU}px;
+              text-align: center;
+            `}
+          >
+            {(!status || status === 'normal' || status === 'loading') && (
+              <img
+                src={publicUrl + illustrationBlueImage}
+                alt=""
+                height={20 * GU}
+                css={`
+                  margin-bottom: ${2 * GU}px;
+                `}
+              />
+            )}
+
+            {(status === 'empty-filters' || status === 'empty-search') && (
+              <img
+                src={publicUrl + illustrationRedImage}
+                alt=""
+                height={20 * GU}
+                css={`
+                  margin-bottom: ${2 * GU}px;
+                `}
+              />
+            )}
+
+            {(!status || status === 'normal') &&
+              (statusEmpty || (
+                <p
+                  css={`
+                    ${textStyle('title2')};
+                  `}
+                >
+                  No data available.
+                </p>
+              ))}
+
+            {status === 'loading' &&
+              (statusLoading || (
+                <p
+                  css={`
+                    ${textStyle('title2')};
+                    display: flex;
+                    align-items: center;
+                  `}
+                >
+                  <LoadingRing
+                    css={`
+                      margin-right: ${2 * GU}px;
+                    `}
+                  />{' '}
+                  Loading data...
+                </p>
+              ))}
+
+            {status === 'empty-filters' && (
+              <>
+                <p
+                  css={`
+                    font-size: 28px;
```

Comment:
Not exactly the same size (26 vs. 28), but we should probably use a text style anyway:

```suggestion
                    ${textStyle('title2')};
```